modules/seqexec/server/src/main/scala/seqexec/server/Header.scala

```diff
@@ -3,33 +3,32 @@
 
 package seqexec.server
 
+import cats.effect.IO
 import seqexec.model.dhs.ImageFileId
 import seqexec.server.keywords._
 
 /**
- * Typeclass for types that can send keywords
- */
+  * Typeclass for types that can send keywords
+  */
 trait HeaderProvider[A] {
-  // Name of the instrument to be sent to the DHS
-  def name(a: A): String
   // Client to send keywords to an appropriate server
-  def keywordsClient(a: A): KeywordsClient
+  def keywordsClient(a: A): KeywordsClient[IO]
 }
 
 object HeaderProvider {
   def apply[A](implicit ev: HeaderProvider[A]): HeaderProvider[A] = ev
 
   final class HeaderProviderOps[A: HeaderProvider](val a: A) {
-    def name: String = HeaderProvider[A].name(a)
-    def keywordsClient: KeywordsClient = HeaderProvider[A].keywordsClient(a)
+    def keywordsClient: KeywordsClient[IO] = HeaderProvider[A].keywordsClient(a)
   }
 
-  implicit def ToHeaderProviderOps[A: HeaderProvider](a: A): HeaderProviderOps[A] = new HeaderProviderOps(a)
+  implicit def ToHeaderProviderOps[A: HeaderProvider](
+      a: A): HeaderProviderOps[A] = new HeaderProviderOps(a)
```

Comment:
Kind of a weird place for a newline.

Reply:
This was done by `scalafmt`, it has some quirks